WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModel.cs

```diff
@@ -93,23 +94,26 @@ public void OnWalletOpened()
 		{
 			Disposables = Disposables is null ? new CompositeDisposable() : throw new NotSupportedException($"Cannot open {GetType().Name} before closing it.");
 
-			var observed = Observable.Merge(
-				Global.UiConfig.WhenAnyValue(x => x.LurkingWifeMode).Select(_ => Unit.Default),
+			Observable.Merge(
 				Observable.FromEventPattern(Global.WalletService.TransactionProcessor, nameof(Global.WalletService.TransactionProcessor.CoinReceived)).Select(_ => Unit.Default),
 				Observable.FromEventPattern(Global.WalletService.TransactionProcessor, nameof(Global.WalletService.TransactionProcessor.CoinSpent)).Select(_ => Unit.Default))
-				.Throttle(TimeSpan.FromSeconds(1))
-				.ObserveOn(RxApp.MainThreadScheduler);
-
-			observed.Subscribe(_ =>
+				.Throttle(TimeSpan.FromSeconds(0.5))
+				.Merge(Global.UiConfig.WhenAnyValue(x => x.LurkingWifeMode).Select(_ => Unit.Default))
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(_ =>
 				{
-					Money balance = WalletService.Coins.TotalAmount();
-
-					Title = $"{Name} ({(Global.UiConfig.LurkingWifeMode ? "#########" : balance.ToString(false, true))} BTC)";
+					try
+					{
+						Money balance = WalletService.Coins.TotalAmount();
+						Title = $"{Name} ({(Global.UiConfig.LurkingWifeMode ? "#########" : balance.ToString(false, true))} BTC)";
+					}
+					catch (Exception ex)
+					{
+						Logger.LogError(ex);
+					}
 				})
 				.DisposeWith(Disposables);
 
-			observed.Next();
```

Comment:
Why remove this?

Reply:
It is unnecessary. The continuation of the subscription is automatic. We are using the same pattern as everywhere else.

Reply:
Why was that necessary? This 1si the only place where we use this pattern.